kie-server-parent/kie-server-remote/kie-server-rest/kie-server-rest-swagger/src/test/java/org/kie/server/remote/rest/swagger/HistorySwaggerDecorationTest.java

```diff
@@ -0,0 +1,138 @@
+/*
+ * Copyright 2020 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+package org.kie.server.remote.rest.swagger;
+
+import static java.util.Collections.emptyListIterator;
+import static org.hamcrest.CoreMatchers.hasItems;
+import static org.hamcrest.CoreMatchers.not;
+
+import javax.ws.rs.GET;
+import javax.ws.rs.POST;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.kie.server.remote.rest.common.marker.KieServerEndpoint;
+import org.kie.server.remote.rest.common.marker.KieServerEndpoint.EndpointType;
+import org.kie.server.remote.rest.common.marker.KieServerEndpointRequestFilter;
+import org.mockito.InjectMocks;
+import org.mockito.runners.MockitoJUnitRunner;
+
+import io.swagger.models.Operation;
+
+@RunWith(MockitoJUnitRunner.class)
```

Comment:
We don't need to use Mockito here. Please remove

Reply:
removed